packages/webapp/src/containers/Animals/Inventory/index.tsx

```diff
@@ -41,6 +41,11 @@ import { useAnimalsFilterReduxState } from './KPI/useAnimalsFilterReduxState';
 import FloatingContainer from '../../../components/FloatingContainer';
 
 interface AnimalInventoryProps {
+  onSelect?: (newIds: string[]) => void;
+  showActionMenu?: boolean;
+  showKPI?: boolean;
+  showFloaterButton?: boolean;
+  showLinks?: boolean;
```

Comment:
Should we consider lumping these all into one prop? like `view = 'task'` or something?

Reply:
This is done!